packages/build-info/bin.js

```diff
@@ -2,4 +2,4 @@
 
 // This is a workaround for npm issue: https://github.com/npm/cli/issues/2632
 
-import './lib/node/bin.js'
+import './lib/src/node/bin.js'
```

Comment:
```suggestion
import './lib/node/bin.js'
```

Reply:
@danez when specifiying the `rootDir` in the tsconfig it does not create the `lib` folder anymore

Reply:
so I'm going to revert this and having a `src` folder